### PR TITLE
✨ add photos & blog resources

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
# ✨ add photos & blog resources 

Closes issue #14

Be sure to complete and check the following steps:

- [x] My changes were made in the `source-web/src/assets/db` folder.
- [x] The resource(s) is completely or partially free.
- [x] The resource(s) is not already in the database.
- [x] All resources have an id, order, name, link and img.
- [x] The ids and urls of the images are not repeated.

### CODE REVIEW

File: source-web/src/assets/db/photos.json [hunk 1]
```diff
@@ -149,5 +149,15 @@
...
+	},
+	{
+		"id": 17,
+		"order": 8,
+		"name": "Picjumbo",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/f_auto,q_auto/v1/freesets/photos/picjumbo",
+		"link": "https://picjumbo.com/",
+		"license": "Freemium",
+		"licenseLink": "https://picjumbo.com/faq-and-terms/",
+		"licenseDescription": "Free with attribution"
```
Comment (cosmoart) [inline source-web/src/assets/db/photos.json:161]: When using Picjumbo you do not need to give attribution.
image

**Review by cosmoart — CHANGES_REQUESTED** [review]

File: source-web/src/assets/db/blogs.json [hunk 2]
```diff
@@ -1,155 +1,166 @@
...
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1706564504/freesets/blogs/medium",
+		"link": "https://www.medium.com",
+		"tags": ["Community"]
+	},
+	{
+		"id": 18,
+		"order": 60,
+		"name": "Chat.Zulip",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1706564504/freesets/blogs/chat-zulip",
+		"link": "https://www.chat.zulip.org",
```
[the remark this answers is not in the document]
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/blogs.json:139]: my bad it's https://[chat.zulip.org](https://chat.zulip.org/)/.

 I will fix that

File: source-web/src/assets/db/photos.json (hunk 1 above)
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/photos.json:161]: I will change that too

File: source-web/src/assets/db/blogs.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/blogs.json]: fixed

File: source-web/src/assets/db/photos.json (hunk 1 above)
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/photos.json]: fixed

File: source-web/src/assets/db/blogs.json [hunk 3]
```diff
@@ -1,155 +1,166 @@
...
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1706564504/freesets/blogs/medium",
+		"link": "https://www.medium.com",
+		"tags": ["Community"]
+	},
+	{
+		"id": 18,
+		"order": 60,
+		"name": "Chat.Zulip",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1706564504/freesets/blogs/chat-zulip",
+		"link": "https://chat.zulip.org",
```
Comment (cosmoart) [inline source-web/src/assets/db/blogs.json:139]: This site is not about blogs or similar.

**Review by cosmoart — CHANGES_REQUESTED** [review]

**Comment by cosmoart** [comment]
The "icons.json" and "tools.json" files have the problems already mentioned in the previous [pull request](https://github.com/cosmoart/Freesets/pull/13#pullrequestreview-1884155710).

File: source-web/src/assets/db/blogs.json (hunk 3 above)
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/blogs.json:139]: removed

**Review by cosmoart — APPROVED** [review]

**Pull request merged**